system_metrics_collector/src/system_metrics_collector/linux_process_cpu_measurement_node.cpp

```diff
@@ -47,6 +47,13 @@ LinuxProcessCpuMeasurementNode::LinuxProcessCpuMeasurementNode(
 {
 }
 
+LinuxProcessCpuMeasurementNode::LinuxProcessCpuMeasurementNode(
```

Comment:
This constructor is required by the composable API (seems like to via the examples, just confirming here).

Reply:
Yep.